assets/templates/dining_lists/dining_slot_diners.html

```diff
@@ -6,48 +6,169 @@
+                                    <a class="btn btn-primary btn-block" href="
+                                    {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}">
+                                        + Add Diner</a>
+                                {% endif %}
+                            </td>
+                            <td>
+                                {% if can_edit_stats or can_edit_pay %}
+                                    <input type="submit" class="btn btn-block btn-primary" value="Update user stats"/>
+                                {% endif %}
+                            </td>
+                            {% if show_delete_column %}
+                            <td> <!-- Empty space --></td>
+                            {% endif %}
+                        </tr>
+
+                        </tbody>
+                    </table>
                 </div>
             </div>
-        </div>
-        {% if can_delete_all or can_delete_some %}
-            <div class="col-2 col-sm-1 p-0 text-center"><b>Remove</b></div>
-        {% endif %}
-    </div>
-
-    <form method="post" class="col-12 p-0">
-        {% csrf_token %}
-        {% for entry in entries %}
-            <hr class="col-12 m-0">
-            {% include 'dining_lists/snippet_dining_entry.html' %}
-        {% endfor %}
-        <hr class="col-12 mt-0">
-        <div class="row col-12 m-0 p-0 p-md-2 py-2 text-size-3">
-            <div class="{% if can_delete_all or can_delete_some %} col-12 col-sm-11 {% else %} col-12 {% endif %} pl-sm-0">
-                <div class="row pr-2">
-                    {% if user_can_add_others %}
-                        <div class="col-12 col-sm-6 order-1 order-sm-0 p-0 px-sm-2">
-                            <a class="btn btn-primary btn-block" href="
-                            {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}">
-                                + Add Diner</a>
-                        </div>
-                    {% endif %}
+            {% csrf_token %}
+            <div class="row d-flex d-sm-none">
+                <div class="col-12">
                     {% if can_edit_stats or can_edit_pay %}
-                        <input type="submit" class="btn btn-block btn-primary col-12 col-sm-6 mt-0 mb-2
-                            {% if not is_open and not user_can_add_others %} offset-sm-6 {% endif %}" value="Update user stats"/>
+                        <input type="submit" class="btn btn-block btn-primary" value="Update user stats"/>
                     {% endif %}
+                   {% if user_can_add_others %}
+                       <a class="btn btn-primary btn-block" href="
+                        {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug %}">
+                           + Add Diner</a>
```

Comment:
This has already been changed some times but the use of capitals here is inconsistent with other places, e.g. the other button has 'Update user stats' and not 'Update User Stats'.

Reply:
The capitalisation is very inconsistent flaw in my process, yeah... haha

Reply:
It's still inconsistent with the rest of the app after your change :wink: